StarterProject/specs/productpage.spec.js

```diff
@@ -0,0 +1,52 @@
+
+import Page from '../pageobjects/page.js';
+import Common from '../pageobjects/common.js';
+import FeaturedProductPage from '../pageobjects/featuredProductPage.js';
+
+const chai = require('chai');
+const should = chai.should();
+const expect = chai.expect;
```

Comment:
General comment about mixing assertion libraries -- we should probably just pick one and go with it 😄  They're all pretty much the same, and are more about readability preferences